tests/utils.py

```diff
@@ -48,7 +48,7 @@ def load_all(self):
                 ]
             },
             "packages": {
-                "include": ["package-name-.*", "gcc.*", "httpd.src", "pkg-debuginfo.*"],
```

Comment:
@rbikar, I removed this as it seems to conflict with the similar item in `exclude`. Please lmk if there's some nuance that I might be missing.

context: I tried to include a package, "package-name-a.foo", which matched the whitelisted pattern but also failed (matched) the blacklist check.